src/main/java/com/jhuguet/sb_taskv1/app/BootUpInfo.java

```diff
@@ -59,14 +56,26 @@ private void addCertificate(int i) {
     }
 
     GiftCertificate buildGiftCertificate(int i) {
-        String localDate = LocalDateTime.now().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
-        return GiftCertificate.builder()
+        String localDate = LocalDateTime
+                .now()
+                .format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
+        return GiftCertificate
+                .builder()
                 .name("Certificate" + i)
                 .description("Description" + i)
                 .price(BigDecimal.valueOf(10.00))
                 .duration(10)
                 .createDate(localDate)
                 .lastUpdateDate(localDate)
-                .associatedTags(new HashSet<>()).build();
+                .associatedTags(new HashSet<>())
```

Comment:
I think you can init tags in constructor of GiftCertificate.

Reply:
Aren't we using builder to avoid using constructors?